app/Jobs/RequeuePendingQsBatchesJob.php

```diff
@@ -48,10 +51,11 @@ private function requeueStalledBatches(): void
         QsBatch::where([
             ['pending_since', '<>', null],
             ['pending_since', '<', $threshold],
+            ['failed', '=', false],
         ])
             ->increment(
                 'processing_attempts', 1,
-                ['pending_since' => null, 'done' => 0]
+                ['pending_since' => null, 'done' => false]
```

Comment:
This is just a stylistic change so that the class always uses `true|false` instead of `true|false|0|1`